features/fetch-crl/sl6/config.pan

```diff
@@ -0,0 +1,15 @@
+unique template features/fetch-crl/sl6/config;
+
+include "components/chkconfig/config";
+"/software/components/chkconfig/service" = {
+    # Run fetch-crl on boot
+    SELF[escape('fetch-crl-boot')] = dict("on", "",
+                                        "startstop", true);
+
+    # Enable periodic fetch-crl (cron)
+    SELF[escape('fetch-crl-cron')] = dict("on", "",
+                                        "startstop", true);
+
+    SELF;
+};
```

Comment:
IMHO this would be much nicer to look at and probably a better example for people to follow:

```pan
"/software/components/chkconfig/service/{fetch-crl-boot}" = dict(
    "on", "",
    "startstop", true,
);

"/software/components/chkconfig/service/{fetch-crl-cron}" = dict(
    "on", "",
    "startstop", true,
);
```

Reply:
Indeed! I just copied/pasted the original code in this template but you re right this is a good occasion to improve it!